src/eepp/ui/doc/syntaxdefinitionmanager.cpp

```diff
@@ -775,6 +775,130 @@ static void addGo() {
 		  "//" } );
 }
 
+static void addAdept() {
```

Comment:
Can you move it to its own file just like most of the other languages? I'm going to move *all* the langs to its own file, this is not the case currently because previously all was in a single file and I got too lazy to move them all, but I will.